container_workflow_tool/git_operations.py

```diff
@@ -130,25 +144,23 @@ def is_git(x): return os.path.isdir(os.path.join(x, '.git'))
         if not components:
             # Get the whole subdirectory
             files = [f.path for f in os.scandir(tmp) if is_git(f.path)]
-        elif isinstance(components, list):
-            files = [path for path in [os.path.join(tmp, c) for c in components] if is_git(path)]
-        elif isinstance(components, str) and is_git(os.path.join(tmp, components)):
-            files = [os.path.join(tmp, components)]
         else:
-            raise u.RebuilderError("Unknown component: {}".format(str(components)))
+            if isinstance(components, str):
+                components = [components]
+            files = [path for path in [os.path.join(tmp, c) for c in components] if is_git(path)]
         if not files:
             self.logger.warn("No git repositories found in directory " + tmp)
         # Walk through the repositories and show changes made in the last commit
         for path in files:
             repo = Repo(path)
             # Only show changes if there are unpushed commits to show
             # or we only want the diff of unstaged changes
-            if self._get_unpushed_commits(repo) or diff:
+            if self.are_unpushed_commits_available(repo) or diff:
                 # Clears the screen
                 print(chr(27) + "[2J")
                 # Force pager for short git diffs
                 subprocess.run(
-                    "git config core.pager 'less -+F' --replace-all",
+                    "git -c \"core.pager='less -+F'\" --replace-all",
```

Comment:
Pretty sure this change will have no effect unless it is actually used in the command on line 168.

Reply:
I see your point. I totally missed the row on line 168.